app/src/main/java/com/theopensourcefamily/chessclock/BoardState.kt

```diff
@@ -0,0 +1,6 @@
+package com.theopensourcefamily.chessclock
+
+sealed class BoardState
```

Comment:
Super unimportant, but as we are not dealing with the physical board in this app, we could choose a name related with the clocks, like ClockState, or CountdownsState, or ChronosState, or Simply MainState (or however we name the activity/fragment + State). What do you think? 🤔

Reply:
Absolutly true! Sorry, I don't know why I named it "board"

Reply:
No need to be sorry, we're here to have fun :D!